tests/functional/dashboard/test_catalogue.py

```diff
@@ -225,6 +225,8 @@ def test_can_create_a_child_product(self):
             'dashboard:catalogue-product-create-child',
             kwargs={'parent_pk': parent_product.pk})
         form = self.get(url).form
+        form['title'] = u'{} - variant 1'.format(parent_product.title)
+        form['upc'] = u'{}-1'.format(parent_product.upc)
```

Comment:
The fix I made here is wrong - there is an underlying issue with the `NullCharField` that we need to fix. I will look into it.